.travis.yml

```diff
@@ -18,6 +19,10 @@ before_install:
 
 install:
   - travis_retry npm install
-  - npm install --no-save --ignore-scripts `npx npm-get-version serverless@$SERVERLESS_VERSION`
+  - travis_retry npm install --no-save --ignore-scripts `npx npm-get-version serverless@$SERVERLESS_VERSION`
+
+script:
+  - npm run lint
```

Comment:
if you adding linter, we should fail CI on not correctly linted code at least

Reply:
yes, CI will fail in current configuration. so I don't see any things to do here.

Reply:
@randomhash could you explain why it won't fail if you see any case..

Reply:
because I saw that lint command will invoke fix command, that should not produce errors ;)